### ISSUE TITLE
[bug] <KFP v2 Metrics Doesn't show up>

### ISSUE THREAD

**Issue by kimkihoon0515** [kubeflow/pipelines#8675]
### Environment

*  How do you deploy Kubeflow Pipelines (KFP)?
I deployed Kubeflow using  this command
```
minikube start --driver=docker --kubernetes-version=1.23.14 --memory=12g --cpus=4
```
*  KFP version:
1.8.18

### Steps to reproduce
I'm trying to get confusion matrix metric output by using kfp v2 like this
```
@component(
    packages_to_install=['scikit-learn'],
    base_image='python:3.9'
)
def iris_sgdclassifier(test_samples_fraction: float, metrics: Output[ClassificationMetrics]):
    from sklearn import datasets, model_selection
    from sklearn.linear_model import SGDClassifier
    from sklearn.metrics import confusion_matrix

    iris_dataset = datasets.load_iris()
    train_x, test_x, train_y, test_y = model_selection.train_test_split(
        iris_dataset['data'], iris_dataset['target'], test_size=test_samples_fraction)


    classifier = SGDClassifier()
    classifier.fit(train_x, train_y)
    predictions = model_selection.cross_val_predict(classifier, train_x, train_y, cv=3)
    metrics.log_confusion_matrix(
        ['Setosa', 'Versicolour', 'Virginica'],
        confusion_matrix(train_y, predictions).tolist() # .tolist() to convert np array to list.
    )

```
and it shows me a confusion matrix output in "Visualizations" tab.
![image](https://user-images.githubusercontent.com/63439911/212465593-fb041328-48eb-4e74-bb1b-41dadc8c3522.png)

But nothing shows up in "Input/Output" tab.
![image](https://user-images.githubusercontent.com/63439911/212465617-d79e1f0a-37df-4300-ba1e-b40a000c998a.png)

### Expected result

Actually I'm trying to compare confusion matrixes of several runs which is possible on kfp v2. 
<img width="749" alt="image" src="https://user-images.githubusercontent.com/63439911/212465776-cbeb5807-4294-4dac-91ba-6c0c52ec750e.png">

the log says 
```
launcher.go:560] Local filepath "/minio/mlpipeline/v2/artifacts/pipeline/v2-metrics/58c160ea-4ef3-49c8-a08a-0e1c0347a031/iris-sgdclassifier/metrics" does not exist
```
I think that log comes out because theres no output file in the component. But idk how to do it.

### Materials and reference
https://www.kubeflow.org/docs/components/pipelines/v1/sdk-v2/run-comparison/

---

Impacted by this bug? Give it a 👍.

**Comment by kimkihoon0515** [comment]
@zijianjoy Hey Thx for your help. I tried kfp version 2.0.0b2 and i got the screen that i wanted :)

**Comment by pythonking6** [comment]
This needs to be re-opened because it is happening in version 2.5.0
